frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/persistent-volume-claim-wrapper.ts

```diff
@@ -44,9 +45,10 @@ export class PersistentVolumeClaimWrapper extends K8sResourceWrapper<
 
   getSize = (): { value: number; unit: string } => {
     const parts = stringValueUnitSplit(getPvcStorageSize(this.data as any) || '');
+    const { value, unit } = humanizeBinaryBytes(parts[0]);
```

Comment:
that looks dangerous, are you sure the unit is always byte (e.g. can you drop part[1] and assume it's always "B") ?